website/docs/best-practices/how-we-mesh/mesh-4-faqs.md

```diff
@@ -0,0 +1,297 @@
+
+</detailsToggle>
+
+<detailsToggle alt_header="Can you have multiple owners in a group?">
+
+No, a [group](/docs/collaborate/govern/model-access#groups) can currently only be assigned to have a single owner. Note however that the assigned can be a team, not just an individual.
+
+</detailsToggle>
+
+<detailsToggle alt_header="Can contracts be assigned individual owners?">
+
+Not directly, but contracts are [assigned to models](/docs/collaborate/govern/model-contracts) and models can be assigned to individual owners. You can use meta fields for this purpose.
+
+</detailsToggle>
+
+<detailsToggle alt_header="Can I make a model “public” only for specific team(s) to use?">
+
+This is not currently possible, but something we hope to enable in the near future. If you’re interested in this functionality, please reach out to your dbt Labs account team.
+
+</detailsToggle>
+
+<detailsToggle alt_header="Is it possible to orchestrate job runs across multiple different projects?">
+
+dbt Cloud will soon offer a capability to trigger jobs on the completion of another job, including a job in a different project. This offers one mechanism for executing a pipeline from start to finish across projects.
+
+</detailsToggle>
+
+## Access and permissions
+
+<detailsToggle alt_header="How do user access permissions work in a dbt Mesh? ">
+
+The existence of projects that have at least one public model will be visible to everyone in the organization with read-only access. 
+
+Private or protected models require a user to have read-only access on the specific project in order to see its existence.
+
+</detailsToggle>
+
+<detailsToggle alt_header="Is it possible to request access permissions from other teams?">
+
+There is not currently! But this is something we may evaluate for the future.
+
+</detailsToggle>
+
+<detailsToggle alt_header="Can projects be hidden for confidentiality?">
+
+The "multi-project" view in dbt Cloud Explorer (and the underlying Discovery API) will include any project that has defined at least one "public" model, including the list of public models. (Read more about [model access modifiers](/docs/collaborate/govern/model-access#access-modifiers).) If a user has additional permissions on that project (managed via dbt Cloud RBAC), including read-only permissions, they can explore further into that project and see details about all models (private, protected, and protected).
+
+</detailsToggle>
+
+<detailsToggle alt_header="As a central data team member, can I still maintain visibility on the entire organizational DAG?">
+
+Yes! As long as a user has permissions (at least read-only access) on all projects in a dbt Cloud account, they can navigate across the entirety of the organization’s DAG in dbt Explorer, and see models at all levels of detail.
+
+</detailsToggle>
+
+<detailsToggle alt_header="TO DO -- How do model-level access, dbt Cloud's role-based access, and data platform access interact?">
```

Comment:
answer needs filling in: https://www.notion.so/WIP-dbt-Mesh-Customer-FAQ-bdadabc9917a4be697db53142c6ff979?pvs=4#4c13ae080f1e4911a1ce915ccb9db5ad